.development/googlesheets_dataclean.R

```diff
@@ -0,0 +1,40 @@
+library(tidyverse)
```

Comment:
Just a note that adding the whole suite of `tidyverse` packages is a lot of overhead as it's not always necessary to use all the packages in `tidyverse`. It's okay to do here as this sits outside the app.

Reply:
Yes, the original intention was that this would just sit outside the app so did not take dependencies etc. into consideration. Since we're adding the packages here to renv, I might replace {tidyverse} with individual packages like {dplyr} and {purrr}. What do you think?

Reply:
Yeah now that you mention it, calling the individual packages will make the installing of packages when running `renv::restore()` faster.